remix.env.d.ts

```diff
@@ -1,2 +1,3 @@
 /// <reference types="@remix-run/dev" />
 /// <reference types="@remix-run/node" />
+/// <reference types="vite/client" />
```

Comment:
Unconvinced this is where this should go, but it solves the ts error for `?url` on css files

Reply:
Let's combine the triple-slash directives like you suggest, but rename the file to `env.d.ts` and update `tsconfig.json`'s `include` field accordingly